css/styles.css

```diff
@@ -0,0 +1,52 @@
+body 	{
+	font-family: cursive;
+	font-size: 18px;
+	margin-top: 0px;
+	margin-right: 0px;
+	background-color: #FCDEBC;
+}
+
+nav	{
+	font-family: serif;
+	font-size:  16px;
+	text-align: right;
+	padding: 5px 15px;
+}
+
+img 	{
+	width: 50%;
+	float: right;
+	margin-left: 20px;
+}
+
+h2 {
+	margin-bottom: 0px;
+	font-style: italic;
+	font-size: 200%;
+	text-align: center;
+}
+
+h1	{
+	margin-top: -10px;
+	margin-bottom: 10px;
+	font-size: 375%;
+	text-align: center;
+} 
+
+h4	{
```

Comment:
@gerriecho Just a heads up... you could have combined the margins into a single declaration like this:
```css
margin: 5px 40px 20px 0;
```